Dockerfile

```diff
@@ -9,8 +9,18 @@ RUN mix local.rebar --force
 RUN mix deps.get
 RUN mix deps.compile
 
+FROM elixir:1.9-alpine
+ENV MIX_ENV="prod"
+RUN mix local.hex --force
+RUN mix local.rebar --force
```

Comment:
Do we need to run these two mix commands, if we're copying everything from `elixir`? Same with `mix compile`, could that be moved to `elixir`?

Reply:
Let me try that

Reply:
`mix compile` is different from `mix deps.compile` so I'll that one